index.html

```diff
@@ -1,301 +1,307 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="portfolio.css" />
-    <title>Nsanzimfura portfolio</title>
-  </head>
-  <body>
-    <header>
-      <nav tabindex="0" class="top-navigation-bar">
-        <h1 tabindex="0" class="header-welcome">Welcome</h1>
-        <ul class="desktop-nav-container">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="portfolio.css" />
+  <title>Nsanzimfura portfolio</title>
+</head>
+<body>
+  <header>
+    <nav class="top-navigation-bar">
+      <h1 class="header-welcome">Welcome</h1>
+      <ul class="desktop-nav-container">
+        <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a href="#about-me">About</a></li>
+        <li class="nav-item"><a href="#contact">Contact</a></li>
+      </ul>
+      <span class="horizontal_bar"></span>
+      <div tabindex="0" class="menu-icon">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+
+    </nav>
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Kindly the gray background and the bottom line that appears in the Figma template. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/202182947-c1a7e681-3243-4ed6-b1c8-757d7b9e4a36.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/202183569-f69917d7-7734-49d8-8e11-8324b9c80823.png)

[OPTIONAL] I think it is a good idea to give a position fixed to the menu this way when you scroll down you will still see the menu. It will help with the user's experience. :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/202184296-5512bf72-9b31-4957-82b3-d5f7e845ff39.png)